src/bindgen/config.rs

```diff
@@ -745,6 +800,8 @@ pub struct Config {
     pub line_length: usize,
     /// The amount of spaces in a tab
     pub tab_width: usize,
+    /// The type of line endings to generate
+    pub line_endings: Option<LineEndingStyle>,
```

Comment:
We could probably just make this `LineEndingStyle`, and initialize it with `LineEndingStyle::default()` below, instead of branching every line-break :)

Reply:
I figured "Option" here implied optional in the toml? Is the toml always overlaid on top of the default values and all keys are optional?

Reply:
Yes, all keys are optional, otherwise your patch wouldn't pass CI :)